recipe/recipe.yaml

```
@@ -0,0 +1,53 @@
+schema_version: 1
+
+context:
+  name: bioio-base
+  version: 1.0.7
+  python_min: 3.10
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/bioio_base-${{ version }}.tar.gz
+  sha256: dfa3c1f9d373ca3a56f66e3694286a3db20dab972d9fe455eebe0d0ddb075a7d
+
+build:
+  number: 3
+  noarch: python >=${{ python_min }}
```

This didn't work, let's undo:
```suggestion
  noarch: python
```